rpcs3/CMakeLists.txt

```diff
@@ -438,7 +439,7 @@ else()
 	if(CMAKE_SYSTEM MATCHES "Linux")
 		target_link_libraries(rpcs3 "${RPCS3_SRC_DIR}/../3rdparty/discord-rpc/lib/libdiscord-rpc-linux.a")
 	elseif(APPLE)
-		target_link_libraries(rpcs3 "${RPCS3_SRC_DIR}/../3rdparty/discord-rpc/lib/libdiscord-rpc-mac.a")
+		target_link_libraries(rpcs3 "${RPCS3_SRC_DIR}/../3rdparty/discord-rpc/lib/libdiscord-rpc-mac.a" objc "-framework Foundation" "-framework CoreServices")
```

Comment:
I'd appreciate it if this line was actually added to the other `-framework` link_libraries on line 421

Reply:
I think these are needed specifically for discord integration, hence I put it this way. I don't mind changing it though.

Reply:
hm, alright it's fine to leave them there then.